### DISCUSSION TITLE
Add an extra Video provider in video component grapesjs

### DISCUSSION THREAD

**Discussion by Mirna-Nasrallah** [GrapesJS/grapesjs#4522] [Q&A] [answered]
I'm currently working on a php pagebuilder website and I tried numerous ways to extend the video component to add another provider but It's not working, any ideas?

**Comment by artf** [comment, marked as answer]
@Mirna-Nasrallah I'd suggest checking out the source of the video component ([Model](https://github.com/artf/grapesjs/blob/master/src/dom_components/model/ComponentVideo.js) - [View](https://github.com/artf/grapesjs/blob/master/src/dom_components/view/ComponentVideoView.js)) this will make you understand what methods you have to extend in order to add your providers

**Reply by Mirna-Nasrallah** [reply]
I'll check that out thank you
